server/tserver/src/main/java/org/apache/accumulo/tserver/replication/AccumuloReplicaSystem.java

```diff
@@ -362,6 +400,7 @@ protected Status replicateRFiles(ClientContext peerContext, final HostAndPort pe
   protected Status replicateLogs(ClientContext peerContext, final HostAndPort peerTserver, final ReplicationTarget target, final Path p, final Status status,
       final long sizeLimit, final String remoteTableId, final TCredentials tcreds, final ReplicaSystemHelper helper, final UserGroupInformation accumuloUgi)
       throws TTransportException, AccumuloException, AccumuloSecurityException {
+    stopReplication.set(false);
```

Comment:
I checked on that and a new instance of `AccumuloReplicaSystem` is created for every task.